lib/adf_web.py

```diff
@@ -653,8 +656,7 @@ def jinja_list(seas_list):
                 #Check if the mean plot type page exists for this case:
```

Comment:
I might change this comment too given that we aren't actually checking if the plot type page exists anymore.